cgmanifests/generated/cgmanifest.json

```diff
@@ -6,7 +6,7 @@
       "component": {
         "type": "git",
         "git": {
-          "commitHash": "d52c46520124845b1e0e0525f2759299d840143f",
```

Comment:
This line of change should be reverted. Your local emsdk submodule wasn't up to date.